CorsixTH/Lua/base_config.lua

```diff
@@ -25,9 +25,12 @@ local configuration = {
   -----------------------------------------------------------
   --      New configuration values added in CorsixTH       --
   -----------------------------------------------------------
+  -- Interest rate and overdraft differential values will be divided by 10,000
```

Comment:
Not your fault, more the original game's... but silly imo :)

Reply:
I suspect the original did this to avoid dealing with floating point. When it comes to multiplayer we may wish we had followed their example.